modules/ROOT/pages/constraints/syntax.adoc

```diff
@@ -206,95 +144,69 @@ Where `<TYPE>` is one of the following property types:
+[source, syntax]
 ----
 CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
 FOR (n:LabelName)
 REQUIRE n.propertyName IS [NODE] KEY
 [OPTIONS "{" option: value[, ...] "}"]
 ----
 
-[source, syntax, role="noheader", indent=0]
+.Syntax for creating a composite node key constraint on multiple properties
+[source, syntax]
 ----
 CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
 FOR (n:LabelName)
 REQUIRE (n.propertyName_1, ..., n.propertyName_n) IS [NODE] KEY
 [OPTIONS "{" option: value[, ...] "}"]
 ----
 
-Index provider can be specified using the `OPTIONS` clause.
-
-[role=label--enterprise-edition label--new-5.7]
-[[constraints-syntax-create-rel-key]]
-=== Relationship key constraint
-
-This command creates a relationship key constraint on relationships with the specified relationship type and properties.
-
-[source, syntax, role="noheader", indent=0]
+.Syntax for creating a relationship key constraint on a single property
+[source, syntax]
 ----
 CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
 FOR ()-"["r:RELATIONSHIP_TYPE"]"-()
 REQUIRE r.propertyName IS [REL[ATIONSHIP]] KEY
 [OPTIONS "{" option: value[, ...] "}"]
 ----
 
-[source, syntax, role="noheader", indent=0]
+.Syntax for creating a composite relationship key constraint on multiple properties
+[source, syntax]
 ----
 CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
 FOR ()-"["r:RELATIONSHIP_TYPE"]"-()
 REQUIRE (r.propertyName_1, ..., r.propertyName_n) IS [REL[ATIONSHIP]] KEY
 [OPTIONS "{" option: value[, ...] "}"]
 ----
 
-Index provider can be specified using the `OPTIONS` clause.
+An index provider can be specified using the `OPTIONS` clause.
 
+For examples on how to create key constraints, see  xref:constraints/managing-constraints.adoc#create-key-constraints[Create, show, and drop constraints -> Create key constraints].
+Key constraints are xref:constraints/managing-constraints.adoc#constraints-and-indexes[index-backed].
 
-[[constraints-syntax-drop]]
-== Syntax for dropping constraints
-
-Dropping a constraint is done by specifying the name of the constraint.
-
-[source, syntax, role="noheader", indent=0]
-----
-DROP CONSTRAINT constraint_name [IF EXISTS]
-----
 
-This drop command is optionally idempotent. This means its default behavior is to throw an error if an attempt is made to drop the same constraint twice.
-With the `IF EXISTS` flag, no error is thrown and nothing happens should the constraint not exist.
-As of Neo4j 5.17, an informational notification is instead returned detailing that the constraint does not exist.
+[[list-constraints]]
+== SHOW CONSTRAINTS
 
 [NOTE]
-====
-Dropping a constraint requires the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/database-administration/#access-control-database-administration-constraints[`DROP CONSTRAINT` privilege].
-====
-
-
-[[constraints-syntax-list]]
-== Syntax for listing constraints
-
-List constraints in the database, either all or filtered on constraint type.
```

Comment:
do we want to remove this sentence? Create doesn't seem to have them but the drop does 🤷